### PR TITLE
feat(portals-admin): Institution application lookup

### PR DESCRIPTION
**Pull request by Halli1414**
# Institution application lookup

Attach a link to issue if relevant

## What

Admin portal for Institutions to look up their own applications from users.
This screen is the same as the current admin application portal with restrictions on which appliactions they can query.
New endpoints for instituition application lookup.
New mapper for Institutions nationalIds, some are missing

## Why

Island.is admin are getting many requests for application look up from institutions, this feature should reduce suchs requests.

## Screenshots / Gifs

image

## Checklist:

- [x] I have performed a self-review of my own code
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] Formatting passes locally with my changes
- [x] I have rebased against main before asking for a review

### CODE REVIEW

**Review by thordurhhh — APPROVED** [review]
service-portal LGTM.

File: libs/application/api/core/src/lib/application/application.service.ts [hunk 1]
```diff
@@ -111,6 +115,58 @@ export class ApplicationService {
...
+    page: number,
+    count: number,
+    status?: string,
+    applicantNationalId?: string,
+    from?: string,
+    to?: string,
+  ): Promise<ApplicationPaginatedResponse> {
+    const statuses = status?.split(',')
+    const typeIds = this.getTypeIdsForInstitution(nationalId)
+    const toDate = to ? endOfDay(new Date(to)) : undefined
```
Comment (obmagnusson) [inline libs/application/api/core/src/lib/application/application.service.ts:129]: No option to filter by a specific Time?

File: apps/application-system/api/src/app/modules/application/tools/applicationAdminPaginated.serializer.ts [hunk 2]
```diff
@@ -0,0 +1,175 @@
+import {
```
Comment (obmagnusson) [inline apps/application-system/api/src/app/modules/application/tools/applicationAdminPaginated.serializer.ts:1]: Should just use the admin one.

**Review by obmagnusson — CHANGES_REQUESTED** [review]
Great awesome well job!! 💯 🎉 . Just some minor nuances and questions really.

File: libs/application/api/core/src/lib/application/application.service.ts (hunk 1 above)
Comment (Halli1414) [reply libs/application/api/core/src/lib/application/application.service.ts]: I was trying to mimic the admin portal where to date is inclusive but you're right, should be done client side and leave the posibility of a specific time filter from the api

File: libs/auth-api-lib/seeders/data/scope-application-system-admin.ts [hunk 3]
```diff
@@ -1,6 +1,18 @@
 import { compose, createScope } from './helpers'
 
 export const up = compose(
+  createScope({
```
Comment (saevarma) [inline libs/auth-api-lib/seeders/data/scope-application-system-admin.ts:4]: Please contact and keep @jonarnarbriem in the loop to verify the configuration in each deployed environment as the seed script might be buggy, it has not been maintained for a while.

**Review by saevarma — APPROVED** [review]
LGTM

**Pull request merged**